src/PowerSystems.jl

```diff
@@ -241,6 +248,7 @@ export ForecastCache
 export StaticTimeSeriesCache
 export NormalizationFactor
 export NormalizationTypes
+export TimeSeriesCounts
```

Comment:
Do we need to export this?

Reply:
We export `get_time_series_counts` which returns an instance of this struct. I don’t think a public function should return a private struct. We could argue whether that function should be exported.